docs/_docs/developer-guide/redis-accounting.md

```
@@ -0,0 +1,578 @@
+nav_order: 101
+parent: Reference
+layout: default
+linkTitle: "Redis-Backed Accounting"
+date: 2026-05-22
+description: >
+  Technical reference for the Redis-backed accounting subsystem shared by Cuebot and the Rust scheduler
+---
+
+# Redis-Backed Accounting Reference
+
+### How Cuebot and the Rust scheduler coordinate per-show resource accounting through Redis
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)